Uplift/Views/Home/FitnessCenterView.swift

```diff
@@ -127,6 +128,56 @@ struct FitnessCenterView: View {
         .frame(minWidth: 84, alignment: .leading)
     }
 
+    private var equipmentSection: some View {
+        VStack(spacing: 12) {
+            sectionHeader(text: "EQUIPMENT")
+
+            ScrollView(.horizontal) {
+                HStack(spacing: 12) {
+                    ForEach(fc?.equipment.allTypes() ?? [], id: \.self) { equipmentType in
+                        VStack(alignment: .leading, spacing: 4) {
+                            Text(equipmentType.description)
+                                .lineLimit(1)
+                                .font(Constants.Fonts.bodySemibold)
+                                .padding(EdgeInsets(top: 16, leading: 16, bottom: 2, trailing: 16))
+
+                            equipmentTypeCellView(eqmtType: equipmentType)
+                                .frame(alignment: .leading)
+
+                            Spacer()
+                        }
+                        .frame(width: 247)
+                        .fixedSize(horizontal: true, vertical: false)
+                        .overlay(
+                            RoundedRectangle(cornerRadius: 12)
+                                .stroke(Constants.Colors.gray01, lineWidth: 1)
+                                .upliftShadow(Constants.Shadows.smallLight)
+                        )
```

Comment:
I would factor this out into another supporting view helper function, similar to the one you created below.